src/authservice/chart/values.yaml

```diff
@@ -0,0 +1,162 @@
+  # authorization_uri: ""
+  # Global Token URI Value to set if not using Keycloak
+  # token_uri: ""
+  # Default client_id to be used in each chain
+  client_id: "global_id"
+  # -- default client_secret to be used in each chain
+  client_secret: "global_secret"
+  match:
+    # -- Header to match.  The value ":authority" is used to match the requested hostname
+    header: ":authority"
+    # -- value matches the start of the header value defined above
+    prefix: "uds"
+  # -- Logout URL for the client
+  logout_path: "/globallogout"
+  # -- Logout Redirect URI for the client
+  logout_redirect_uri: ""
+  absolute_session_timeout: 0
+  idle_session_timeout: 0
+  # -- CA signing the OIDC provider. Passed through as a Helm multi-line string. See README for example.
+  certificate_authority: ""
+  # certificate_authority: |
+  #   -----BEGIN CERTIFICATE-----
+  #   MIIE4jCCAsqgAwIBAgIBATANBgkqhkiG9w0BAQsFADARMQ8wDQYDVQQDEwZzZm8t
+  #   Y2EwHhcNMTkxMjIwMDAxNjI1WhcNMjEwNjIwMDAxNjIxWjARMQ8wDQYDVQQDEwZz
+  #   -----END CERTIFICATE-----
+
+  # -- URI for Redis instance used for OIDC token storage/retrieval. This may also be specified per-chain.
+  # redis_server_uri: tcp://{{ .Release.Name }}-{{ .Release.Namespace }}-auth-redis-master:6379/
+  oidc:
+    # -- OpenID Connect hostname.  Assumption of Keycloak based on URL construction
+    host: login.uds.dev
+    # -- Realm for OpenID Connect
+    realm: doug
+  # -- escaped json for the JWKS
+  jwks: ""
+  # jwks: "{\"keys\":[{\"kid\":\"4CK69bW66HE2wph9VuBs0fTc1MaETSTpU1iflEkBHR4\",\"kty\":\"RSA\",\"alg\":\"RS256\",\"use\":\"sig\",\"n\":\"hiML1kjw-sw25BgaZI1AyfgcCRBPJKPE-wwttqa7NNxptr_5RCBGuJXqDyo3p1vjcbb8KjdKnXI7kWer8b2Pz_RP1m_QcPrKOxSluk7GZF8ARsc6FPGbzYgi8o8cBVSsaml6HZzpN3ZnH4DFZ27ifM-Ul_PyMxZ2aweohIaizXp-rgF7Rqpav5NXUwmcSyH8LP92NVIuFlD3HYTDGosVbfA_u_H25Z4XCGKW_vLDTNrl8PcA3HqIoD-vNavysdxAq_KNw7iLLc0KLsjFYSdJL_54H7QubsGR0AyIrLLurJbqAtvttGJK38k5XYWKIwYGtu6iiJwjSb7UtonVdPh8Vw\",\"e\":\"AQAB\",\"x5c\":[\"MIICoTCCAYkCBgFyLIEqUjANBgkqhkiG9w0BAQsFADAUMRIwEAYDVQQDDAliYWJ5LXlvZGEwHhcNMjAwNTE5MTAzNDIyWhcNMzAwNTE5MTAzNjAyWjAUMRIwEAYDVQQDDAliYWJ5LXlvZGEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCGIwvWSPD6zDbkGBpkjUDJ+BwJEE8ko8T7DC22prs03Gm2v/lEIEa4leoPKjenW+NxtvwqN0qdcjuRZ6vxvY/P9E/Wb9Bw+so7FKW6TsZkXwBGxzoU8ZvNiCLyjxwFVKxqaXodnOk3dmcfgMVnbuJ8z5SX8/IzFnZrB6iEhqLNen6uAXtGqlq/k1dTCZxLIfws/3Y1Ui4WUPcdhMMaixVt8D+78fblnhcIYpb+8sNM2uXw9wDceoigP681q/Kx3ECr8o3DuIstzQouyMVhJ0kv/ngftC5uwZHQDIissu6sluoC2+20YkrfyTldhYojBga27qKInCNJvtS2idV0+HxXAgMBAAEwDQYJKoZIhvcNAQELBQADggEBAIVkoDYkM6ryBcuchdAL5OmyKbmmY4WDrMlatfa3uniK5jvFXrmVaJ3rcu0apdY/NhBeLSOLFVlC5w1QroGUhWm0EjAA4zyuU63Pk0sro0vyHrxztBrGPQrGXI3kjXEssaehZZvYP4b9VtYpus6oGP6bTmaDw94Zu+WrDsWdFs+27VEYwBuU0D6E+ENDGlfR+9ADEW53t6H2M3H0VsOtbArEutYgb4gmQcOIBygC7L1tGJ4IqbnhTYLh9DMKNklU+tq8TMHacps9FxELpeAib3O0J0E5zYXdraQobCCe+ao1Y7sA/wqcGQBCVuoFgty7Y37nNL7LMvygcafgqVDqw5U=\"],\"x5t\":\"mxFIwx7EdgxyC3Y6ODLx8yr8Bx8\",\"x5t#S256\":\"SdT7ScKVOnBW6qs_MuYdTGVtMGwYK_-nmQF9a_8lXco\"}]}"
```

Comment:
not sure I love leaving these here, even if it does show an example of how to use it

Reply:
I'll go back through and clean up some of these example values. Definitely a bit messy and unnecessary.

Reply:
sounds good, def a tomorrow us problem

Reply:
Removed some of the example values where they didn't make sense - especially with this one I think we'd likely be using `jwks_uri` so less concerned about an example for `jwks` - might make sense to drop down that down the road even.

I also dropped a few extra knobs that are for wiring authservice to non-keycloak endpoints (Google SSO, etc) which I don't think we need? I can restore if we want those for something but think dropping them fits with our opinionation on uds-core?